Modules/_ctypes/_ctypes.c

```diff
@@ -5847,20 +5840,25 @@ _ctypes_mod_exec(PyObject *mod)
     return 0;
 }
 
+static struct PyModuleDef_Slot _ctypes_slots[] = {
+    {Py_mod_exec, _ctypes_mod_exec},
```

Comment:
I am just curious about those codes in `_ctypes_mod_exec`, do we need judge the attr manually?
```
_unpickle = PyObject_GetAttrString(mod, "_unpickle");
    if (_unpickle == NULL) {
        return -1;
    }
```